src/main/java/com/mraof/minestuck/client/gui/toasts/GristToast.java

```diff
@@ -0,0 +1,165 @@
+package com.mraof.minestuck.client.gui.toasts;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.*;
+import com.mraof.minestuck.alchemy.GristType;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.components.toasts.Toast;
+import net.minecraft.client.gui.components.toasts.ToastComponent;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.util.Mth;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+@OnlyIn(Dist.CLIENT)
```

Comment:
These annotations are not meant for mod code. Remove this.

Reply:
Should be gone in the latest commits.